Assets/GothicVR/Scripts/Bootstrap/PhoenixBootstrapper.cs

```diff
@@ -1,31 +1,33 @@
-using System;
 using System.Diagnostics;
 using System.IO;
 using AOT;
 using GVR.Creator;
 using GVR.Debugging;
-using GVR.Demo;
 using GVR.Manager;
 using GVR.Manager.Settings;
 using GVR.Phoenix.Interface;
 using GVR.Phoenix.Interface.Vm;
 using GVR.Util;
 using PxCs.Helper;
 using PxCs.Interface;
-using TMPro;
-using Unity.VisualScripting;
-using UnityEngine.TextCore.LowLevel;
+using UnityEngine.XR;
+using UnityEngine;
 using Debug = UnityEngine.Debug;
 
 namespace GVR.Bootstrap
 {
     public class PhoenixBootstrapper : SingletonBehaviour<PhoenixBootstrapper>
     {
         private bool _loaded = false;
+        public GameObject configurationMessage;
+        public GameObject filePickerButton;
```

Comment:
These two aren't set in Bootstrap scene. I get a nullpointer exception.
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/6ad57972-93ea-458e-a9bc-1bfbb7e79e8f)